llama_stack/apis/inference/inference.py

```diff
@@ -319,6 +320,7 @@ class ChatCompletionRequest(BaseModel):
     response_format: Optional[ResponseFormat] = None
     stream: Optional[bool] = False
     logprobs: Optional[LogProbConfig] = None
+    response_output_parser: Optional[ResponseOutputParser] = Field(default=ResponseOutputParser.default)
```

Comment:
Synced offline w/ @ashwinb @hardikjshah , we will hold off this and put parsers on client SDK side.